presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -423,6 +423,55 @@ If your workload experiences the error *Timeout waiting for connection from
 pool*, increase the value of both ``hive.s3select-pushdown.max-connections`` and
 the maximum connections configuration for the file system you are using.
 
+Alluxio Configuration
+---------------------
+
+`Alluxio <https://www.alluxio.io/?utm_source=prestodb&utm_medium=prestodocs>`_
+is world’s first open source data orchestration technology for analytics and AI for
+the cloud. It bridges the gap between data driven applications and storage systems, by
+bringing data from the storage tier closer to the data driven applications, and makes it
+easily accessible by enabling applications to connect to numerous storage systems through
+a common interface.
+
+Presto can read and write tables stored in the Alluxio Data Orchestration System
+`Alluxio <https://www.alluxio.io/?utm_source=prestodb&utm_medium=prestodocs>`_,
+leveraging Alluxio's distributed block-level read/write caching functionality.
+The tables must be created in the Hive metastore with the ``alluxio://`` location prefix
+(see `Running Apache Hive with Alluxio <https://docs.alluxio.io/os/user/2.1/en/compute/Hive.html>`_
+for details and examples).
+Presto queries will then transparently retrieve and cache files
+or objects from a variety of disparate storage systems including HDFS and S3.
+
+Alluxio Client-Side Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To configure Alluxio client-side properties on Presto, append the Alluxio
+configuration directory (``${ALLUXIO_HOME}/conf``) to the Presto JVM classpath,
+so that the Alluxio properties file ``alluxio-site.properties`` can be loaded as a resource.
+Update the Presto :ref:`presto_jvm_config` file ``etc/jvm.config`` to include the following:
+
+.. code-block:: none
+
+  -Xbootclasspath/a:<path-to-alluxio-conf>
+
+The advantage of this approach is that all the Alluxio properties are set in
+the single ``alluxio-site.properties`` file. For details, see `Customize Alluxio User Properties
+<https://docs.alluxio.io/os/user/2.1/en/compute/Presto.html#customize-alluxio-user-properties>`_.
+
+Alternatively, add Alluxio configuration properties to the Hadoop configuration
+files (``core-site.xml``, ``hdfs-site.xml``) and configure the Hive connector
+to use the `Hadoop configuration files <#hdfs-configuration>`__ via the
+``hive.config.resources`` connector property.
+
+Deploy Alluxio with Presto
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To achieve the best performance running Presto on Alluxio, it is recommended
+to collocate Presto workers with Alluxio workers. This allows reads and writes
+to bypass the network (*short-circuit*). See `Performance Tuning Tips for Presto with Alluxio
```

Comment:
I'm confused about what the parenthetical "short-circuit" is adding here.

Reply:
Done.